spec/book_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './book'
+require './item'
+
+describe Book do
+  context 'Pass parameters through book class' do
+    new_book = Book.new(11, 'Abubakar', 'Morrison')
+    new_book_two = Book.new(11, 'Abubakar', 'bad')
+    new_book_three = Book.new(2, 'Abubakar', 'Hello')
+
```

Comment:
- [ ] Cover state should be 'good' or 'bad' based on the requirements. Kindly implement this. 👍